samples/bluetooth/llpm/src/main.c

```diff
@@ -409,6 +399,24 @@ static void test_run(void)
 		}
 
 		memset(&llpm_latency, 0, sizeof(llpm_latency));
+		if (conn_info.role == BT_CONN_ROLE_CENTRAL) {
+			static int counter;
+			static uint16_t interval_us = INTERVAL_LLPM_US;
+
+			counter++;
+			if (counter == 20) {
```

Comment:
Why 20? Could you explain in comment or use define instead of magic number?

Reply:
addded comment, but open t changing it some other way. like not doing this switching between intervals even.